plugins/spu2-x/src/Windows/SndOut_XAudio2.inl

```diff
@@ -132,14 +132,13 @@ private:
             pSourceVoice->FlushSourceBuffers();
             pSourceVoice->Start(0, 0);
 
-            qbuffer = new s16[m_nBuffers * m_BufferSize];
-            ZeroMemory(qbuffer, m_BufferSizeBytes * m_nBuffers);
+            m_buffer = std::make_unique<s16[]>(m_nBuffers * m_BufferSize);
```

Comment:
No, it's not Windows specific. See http://stackoverflow.com/questions/42140212/does-make-unique-value-initializes-char-array